README.md

```diff
@@ -9,7 +9,7 @@ A faster implementation of [librsync](https://github.com/librsync/librsync) in
 pure Rust, using SIMD operations where available. Note that only the legacy MD4
```

Comment:
Is this change correct? If so, I recommend making a separate PR directly to upstream rather than changing it here.

Reply:
Sorry for the change..I dont remember the reason..But it is an error